### ISSUE TITLE
[Bug]: MO oom during stability test

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#4845]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.6.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

The mo was killed by oom when executing stability test
the test is as followings:
1、bvt test loop run, 1000 times
2、tpch（0.5G） query loop run ，1000 times
3、tp mixed cases run，20 hours

and nearly  12 hours  after test start，the mo oom：
[74218.421881] [ 559341]     0 559341    19144      225   303104        0             0 sshd
[74218.421884] [ 559344]     0 559344     8586       39   151552        0             0 sshd
[74218.421886] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0-1,global_oom,task_memcg=/user.slice/user-1029.slice/session-16.scope,task=mo-service,pid=68923,uid=1029
[74218.422002] Out of memory: Killed process 68923 (mo-service) total-vm:277634136kB, anon-rss:251547972kB, file-rss:0kB, shmem-rss:0kB, UID:1029 pgtables:530216kB oom_score_adj:0
[74223.476360] oom_reaper: reaped process 68923 (mo-service), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB

**Comment by yingfeng** [comment]
Remove S0 temporarily and need to reassign after restarting

**Comment by nnsgmsone** [comment]
same as #7613

**Comment by aressu1985** [comment]
tracking for one more day

**Comment by nnsgmsone** [comment]
no process

**Comment by nnsgmsone** [comment]
no process

**Comment by nnsgmsone** [comment]
no process

**Comment by nnsgmsone** [comment]
no process

**Comment by nnsgmsone** [comment]
no process

**Comment by nnsgmsone** [comment]
I will give a plan to take care of the memory problem.

**Comment by fengttt** [comment]
Taking this off from S-1.   @nnsgmsone has his grand plan of addressing this in a general, systematic way.

**Comment by aressu1985** [comment]
I closed this , and tracked it with #8327
